mlrun/api/api/utils.py

```diff
@@ -679,6 +679,13 @@ def _submit_run(
         # fn.spec.rundb = "http://mlrun-api:8080"
         schedule = data.get("schedule")
         if schedule:
+            if data.get("overwrite_schedule"):
+                get_scheduler().delete_schedule(
```

Comment:
if it failed deleting the schedule, what error message will be printed?

Reply:
added 'safe' delete + warning message.